src/css/index.scss

```diff
@@ -1,2 +1,167 @@
 @import './global';
 @import './information';
+
+$basic-blue: #bcc8dc;
+$basic-grey: #edeff5;
+$dark-red: #d93347;
+$dark-blue: #314a74;
+
+.HOME {
+  position: fixed;
+  left: 70px;
+  top: 68px;
+  color: white;
+  font-size: 20px;
+  font-weight: bold;
+}
+.HOME:hover {
+  color: $basic-blue;
+}
+
+header {
+  height: 200px;
+}
+
+section ul {
+  height: 440px;
+  width: 948px;
+  position: relative;
+  left: 220px;
+  top: 230px;
+  align-items: flex-start;
+  align-content: flex-start;
+}
+.wrapper-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  font-weight: bold;
+  text-align: center;
+  position: relative;
+  left: 360px;
+  top: 130px;
+}
+
+.wrapper-box > * {
+  padding: 20px;
+  flex-grow: 0;
+  vertical-align: middle;
+}
+
+//Skills Name and Title
+.dark_red_font {
```

Comment:
If you used the kebab class naming `.wrapper-box`, just keep the class naming consistent